### ISSUE TITLE
can we get spago 0.93 in here, somehow?

### ISSUE THREAD

**Issue by silky** [justinwoo/easy-purescript-nix#232]
that'd be great! :pray:

**Comment by justinwoo** [comment]
sounds like you'll have to be the one to ask for things:

1. according to https://github.com/purescript/spago#installation, there should be a release for that version, but there isn't.
2. once you successfully get a release for the upstream project, you can simply fork this repo and modify this file: https://github.com/justinwoo/easy-purescript-nix/blob/master/spago.nix

**Comment by silky** [comment]
yes; and i suppose i'll have to modify spago2nix as well, given spago has changed it's package-listing format from dhall to yaml

🥲 🥲 🥲 🥲 🥲

**Comment by silky** [comment]
this fork has it - https://github.com/f-f/easy-purescript-nix - maybe it could be contributed back @f-f ?

**Comment by f-f** [comment]
I meant to PR it here once we declare it an alpha version (should happen some time at the end of the month), right now there's not much documentation so I wouldn't want to encourage widespread use

**Comment by mstksg** [comment]
@f-f wondering if there has been any updates on this, or if there's a stable fork you might recommend using :)

**Comment by f-f** [comment]
@mstksg I have switched all my projects to [purescript-overlay](https://github.com/thomashoneyman/purescript-overlay), which has spago-next and is kept up to date automatically whenever there's a new release
